soc/arm/nuvoton_npcx/common/soc_dt.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2020 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _NUVOTON_NPCX_SOC_DT_H_
+#define _NUVOTON_NPCX_SOC_DT_H_
+
+/* Macro function used foe device tree */
+
+/******************************************************************************/
+/* Devicetree macros for turning on/off clock of device */
+/*
```

Comment:
Should join these two comment blocks

But this macro could use an example as it is unclear how to use it